File: 240805/총합 구하기/find-the-total.c
#include <stdio.h>

int main() {
    int a, b, ans = 0;
    scanf("%d %d", &a, &b);

    for (int i=a; i <=b ;i++) {
        if (i % 6 == 0 && i % 8 != 0) {
            ans += i;
        }
    }
    printf("%d", ans);
    return 0;
}

File: 240805/총합 구하기/find-the-total.js
const fs = require('fs')
const input = fs.readFileSync(0).toString().trim().split(" ").map(x => Number(x))

const [a, b] = input
let ans = 0
for (let i = a; i <= b; i++) {
    if (i % 6 == 0 && i % 8 != 0) {
        ans += i
    }
}

console.log(ans)